python/lsst/ts/linearstage/controllers/igus_dryve/igus_dryve.py

```diff
@@ -1,23 +1,15 @@
 __all__ = ["IgusLinearStageStepper"]
 
 import asyncio
-import time
 import logging
-import numpy as np
+import time
 
+import numpy as np
 from lsst.ts import salobj, utils
-from lsst.ts.LinearStage.mocks.mock_igusDryveController import MockIgusDryveController
-from lsst.ts.LinearStage.controllers.igus_dryve.igus_utils import (
-    read_telegram,
-    derive_handshake,
-    interpret_read_telegram,
-)
-
-from lsst.ts.LinearStage.controllers.igus_dryve.igusDryveTelegrams import (
-    telegrams_write,
-    telegrams_read,
-    # telegrams_read_errs,
-)
+from lsst.ts.linearstage.mocks.mock_igus_dryve_controller import MockIgusDryveController
```

Comment:
Please use a relative import -- it reduces the danger of circular dependencies that break code, and it makes it clearer that the module is part of this package.